bcda/api/v2/api.go

```diff
@@ -0,0 +1,140 @@
+package v2
+
+import (
+	"github.com/CMSgov/bcda-app/bcda/models/postgres"
+
+	"net/http"
+	"time"
+
+	"github.com/go-chi/chi"
+
+	api "github.com/CMSgov/bcda-app/bcda/api"
+	"github.com/CMSgov/bcda-app/bcda/database"
+	"github.com/CMSgov/bcda-app/bcda/models"
+	"github.com/CMSgov/bcda-app/bcda/responseutils"
+	"github.com/CMSgov/bcda-app/bcda/utils"
+)
+
+const (
+	groupAll = "all"
+)
+
+func init() {
```

Comment:
Can we move this `init()` method into bcda/api/requests.go? We want to have the initialization step shared between the v1/v2 API impls.